includes/Modules/Tag_Manager.php

```diff
@@ -638,4 +640,25 @@ private function register_tag() {
 		}
 	}
 
+	/**
+	 * Can the Analytics module insert it's own snippet.
```

Comment:
Similar to JS, the doc block should start with a 3p verb.
```suggestion
	 * Filters whether or not the Analytics module's snippet should be controlled by its `useSnippet` setting.
```